### PR TITLE
Update CSV description and design doc

### PR DESCRIPTION
**Pull request by VaishnaviHire**
## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: bundle/manifests/opendatahub-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -202,6 +205,9 @@ spec:
         displayName: Conditions
         path: conditions
       version: v1alpha1
+    - kind: KfDef
```
Comment (zdtsw) [inline bundle/manifests/opendatahub-operator.clusterserviceversion.yaml:208]: need remove

**Review by etirelli — APPROVED** [review]
lgtm

File: bundle/manifests/opendatahub-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (VaishnaviHire) [reply bundle/manifests/opendatahub-operator.clusterserviceversion.yaml:208]: It gets auto generated when I do `make bundle`

**Review by zdtsw — APPROVED** [review]
/lgtm

**Pull request merged**